resources/lang/en/tasks.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+return [
+    'create' => [
+        'title' => 'New Task',
+        'name' => 'Task',
+        'button' => 'Add Task'
```

Comment:
```suggestion
        'button' => 'Add Task',
```
sửa những chỗ tương tự

Reply:
vâng ạ